docs/getting-started/beaglebone.md

```diff
@@ -15,6 +15,13 @@ If you want the latest Node-RED 1.x then you need to use the Alpha Testing - Deb
 
     sudo apt update && sudo apt full-upgrade
 
+Currently, Debian (10) Buster is only available as an SD card image. If flashing the image to the eMMC is desired, the line
+
+    cmdline=init=/opt/scripts/tools/eMMC/init-eMMC-flasher-v3.sh
+
+should be uncommented in the /boot/uEnv.txt file. The image can then be flashed to the eMMC like any 'flasher' image on the
+BeagleBoard website.
```

Comment:
I thought you had to hold down a button or something ?

Reply:
That is correct for flashing an image to a BeagleBone eMMC and it could be included, but that is not unique to this image or method. After the file is modified, the method to flash to the eMMC is the same as for any flasher image available from the BeagleBone latest images site.

Reply:
for the sake of one line of test it may as well be included.

Reply:
I've added instructions in the latest commit.

Reply:
Perfect. Thanks